index.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
 var gutil = require('gulp-util');
+var fs = require('fs');
```

Comment:
That is a leftover from a horrible workaround for [an issue I found a few days ago](https://github.com/sindresorhus/gulp-chown/issues/5).
I'll remove that and re-submit the PR.